src/main/java/org/cascadebot/cascadebot/commandmeta/Module.java

```diff
@@ -33,6 +33,10 @@
     /**
      * All our special commands :D
      */
+    TAGS,
+    /**
+     * This is a temporary module for housing tags whilst in development.
+     */
```

Comment:
Ummmm that should go above not bellow the tags module

Reply:
^ this
also I don't think we should have a tags module as the only thing that's gonna be in it is the tags command. Pretty pointless imo